models/database.go

```diff
@@ -49,8 +46,8 @@ var databaseSchema = [][]string{
 			machine_id VARCHAR(255),
 			custom_labels TEXT,
 			address VARCHAR(255),
```

Comment:
btw, let's replace `VARCHAR(255)` with just `VARCHAR`. That was required for MySQL, but for PostgreSQL it is extra noise.